2018-06-27 22:16:48.238 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:16:48.241 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:17:48.239 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:17:48.242 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:18:48.239 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:18:48.243 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:19:18.053 11456 DEBUG oslo_concurrency.lockutils [req-de1a1ecd-48ac-4623-b353-87033cfa5bd5 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:19:18.634 11456 DEBUG neutron.agent.dhcp.agent [req-dd9e7f2b-af1b-435f-99dd-dea5d9c28e0d - - - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:19:18.637 11456 DEBUG neutron.agent.linux.utils [req-dd9e7f2b-af1b-435f-99dd-dea5d9c28e0d - - - - -] Unable to access /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:18.638 11456 DEBUG neutron.agent.linux.dhcp [req-dd9e7f2b-af1b-435f-99dd-dea5d9c28e0d - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-83432395-abb8-4ad9-a97c-31272620acc2 on network 83432395-abb8-4ad9-a97c-31272620acc2 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-27 22:19:18.639 11456 DEBUG neutron.agent.linux.dhcp [req-dd9e7f2b-af1b-435f-99dd-dea5d9c28e0d - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-83432395-abb8-4ad9-a97c-31272620acc2 on network 83432395-abb8-4ad9-a97c-31272620acc2 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-27 22:19:22.747 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.768 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.789 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.814 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.832 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.853 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', 'link', 'set', 'tap54feb389-26', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.884 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-o', 'link', 'show', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.903 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 22:19:22.932 11456 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap54feb389-26) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 22:19:22.934 11456 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-27 22:19:22.937 11456 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap54feb389-26) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 22:19:22.940 11456 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'54feb389-264e-4ae8-b0b8-92949a82364a', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:32:11:f6'})), record=tap54feb389-26) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 22:19:22.981 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap54feb389-26', 'address', 'fa:16:3e:32:11:f6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:22.997 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.011 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap54feb389-26', 'netns', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.037 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', 'link', 'set', 'tap54feb389-26', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.054 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', 'link', 'set', 'tap54feb389-26', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.074 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Acquired semaphore "iptables-qdhcp-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:19:23.076 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.104 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.131 11456 DEBUG neutron.agent.linux.iptables_manager [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-27 22:19:23.134 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Releasing semaphore "iptables-qdhcp-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:19:23.137 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-o', 'link', 'show', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.157 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', 'addr', 'show', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.176 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'addr', 'add', '11.179.1.2/24', 'scope', 'global', 'dev', 'tap54feb389-26', 'brd', '11.179.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.193 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.207 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Setting IPv4 gateway for dhcp netns on net 83432395-abb8-4ad9-a97c-31272620acc2 to 11.179.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-27 22:19:23.209 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.179.1.1', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.227 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.244 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Cleaning stale devices for network 83432395-abb8-4ad9-a97c-31272620acc2 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-27 22:19:23.245 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.261 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.282 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Building initial lease file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-27 22:19:23.284 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Done building initial lease file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/leases with contents:
1530217163 fa:16:3e:32:11:f6 11.179.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-27 22:19:23.285 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:19:23.287 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:19:23.294 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Unable to access /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:23.297 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host', '--addn-hosts=/var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap54feb389-26', '--dhcp-range=set:tag0,11.179.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:23.332 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:23.333 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:23.334 11456 DEBUG neutron.agent.linux.external_process [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:19:23.335 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:19:23.336 11456 DEBUG oslo_concurrency.lockutils [req-de1a1ecd-48ac-4623-b353-87033cfa5bd5 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:19:24.013 11456 DEBUG neutron.agent.dhcp.agent [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:19:24.015 11456 DEBUG neutron.agent.linux.dhcp [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:19:24.021 11456 DEBUG neutron.agent.linux.dhcp [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:19:24.026 11456 DEBUG neutron.agent.linux.utils [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:24.044 11456 DEBUG neutron.agent.linux.dhcp [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:19:24.046 11456 DEBUG neutron.agent.linux.utils [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:24.062 11456 DEBUG neutron.agent.linux.utils [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:24.082 11456 DEBUG neutron.agent.linux.utils [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:24.083 11456 DEBUG neutron.agent.linux.utils [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:24.085 11456 DEBUG neutron.agent.linux.external_process [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:19:24.086 11456 DEBUG oslo_concurrency.lockutils [req-ec0a9e95-11d9-426e-929f-117a85b7d6d3 - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:19:24.088 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:19:24.090 11456 DEBUG neutron.agent.dhcp.agent [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T20:19:19Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-83432395-abb8-4ad9-a97c-31272620acc2, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.2'}], id=54feb389-264e-4ae8-b0b8-92949a82364a, mac_address=fa:16:3e:32:11:f6, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=False, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:19:20Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-27 22:19:24.091 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:19:24.093 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:19:24.094 11456 INFO neutron.agent.dhcp.agent [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T20:19:19Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-83432395-abb8-4ad9-a97c-31272620acc2, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.2'}], id=54feb389-264e-4ae8-b0b8-92949a82364a, mac_address=fa:16:3e:32:11:f6, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=False, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:19:22Z
2018-06-27 22:19:24.097 11456 DEBUG neutron.agent.dhcp.agent [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:19:24.099 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:19:24.102 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:19:24.108 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:24.128 11456 DEBUG neutron.agent.linux.dhcp [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:19:24.130 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:24.154 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:19:24.174 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:24.175 11456 DEBUG neutron.agent.linux.utils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:19:24.176 11456 DEBUG neutron.agent.linux.external_process [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:19:24.177 11456 DEBUG oslo_concurrency.lockutils [req-a1777913-332d-4e82-ac92-b7f6afefd61d - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:19:48.240 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:19:48.244 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:20:02.447 11456 DEBUG oslo_concurrency.lockutils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:02.448 11456 INFO neutron.agent.dhcp.agent [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T20:20:01Z, description=, device_id=ae428768-b0ee-4e58-bbdd-04bb6bf567c4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.1'}], id=100f0eb2-ece2-4bab-90cb-38f9936d58ee, mac_address=fa:16:3e:92:3b:35, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=False, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:01Z
2018-06-27 22:20:02.450 11456 DEBUG neutron.agent.dhcp.agent [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:02.452 11456 DEBUG neutron.agent.linux.dhcp [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:02.455 11456 DEBUG neutron.agent.linux.dhcp [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:02.460 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:02.477 11456 DEBUG neutron.agent.linux.dhcp [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:02.479 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:02.497 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:02.510 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:02.511 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:02.512 11456 DEBUG neutron.agent.linux.external_process [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:02.513 11456 DEBUG oslo_concurrency.lockutils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:03.552 11456 DEBUG oslo_concurrency.lockutils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:03.553 11456 INFO neutron.agent.dhcp.agent [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T20:20:01Z, description=, device_id=ae428768-b0ee-4e58-bbdd-04bb6bf567c4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.1'}], id=100f0eb2-ece2-4bab-90cb-38f9936d58ee, mac_address=fa:16:3e:92:3b:35, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=False, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:01Z
2018-06-27 22:20:03.555 11456 DEBUG neutron.agent.dhcp.agent [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:03.556 11456 DEBUG neutron.agent.linux.dhcp [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:03.558 11456 DEBUG neutron.agent.linux.dhcp [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:03.564 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:03.582 11456 DEBUG neutron.agent.linux.dhcp [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:03.584 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:03.595 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:03.613 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:03.615 11456 DEBUG neutron.agent.linux.utils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:03.616 11456 DEBUG neutron.agent.linux.external_process [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:03.618 11456 DEBUG oslo_concurrency.lockutils [req-7fe290e6-2909-44e1-b498-803a3f4dff68 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:07.191 11456 DEBUG oslo_concurrency.lockutils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:07.192 11456 INFO neutron.agent.dhcp.agent [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T20:20:01Z, description=, device_id=ae428768-b0ee-4e58-bbdd-04bb6bf567c4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.1'}], id=100f0eb2-ece2-4bab-90cb-38f9936d58ee, mac_address=fa:16:3e:92:3b:35, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=False, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:02Z
2018-06-27 22:20:07.194 11456 DEBUG neutron.agent.dhcp.agent [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:07.196 11456 DEBUG neutron.agent.linux.dhcp [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:07.199 11456 DEBUG neutron.agent.linux.dhcp [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:07.205 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:07.223 11456 DEBUG neutron.agent.linux.dhcp [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:07.225 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:07.241 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:07.255 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:07.256 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:07.256 11456 DEBUG neutron.agent.linux.external_process [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:07.257 11456 DEBUG oslo_concurrency.lockutils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:08.228 11456 DEBUG oslo_concurrency.lockutils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:08.229 11456 INFO neutron.agent.dhcp.agent [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T20:20:01Z, description=, device_id=ae428768-b0ee-4e58-bbdd-04bb6bf567c4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.1'}], id=100f0eb2-ece2-4bab-90cb-38f9936d58ee, mac_address=fa:16:3e:92:3b:35, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=False, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:07Z
2018-06-27 22:20:08.231 11456 DEBUG neutron.agent.dhcp.agent [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:08.233 11456 DEBUG neutron.agent.linux.dhcp [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:08.235 11456 DEBUG neutron.agent.linux.dhcp [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:08.240 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:08.256 11456 DEBUG neutron.agent.linux.dhcp [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:08.258 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:08.272 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:08.289 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:08.290 11456 DEBUG neutron.agent.linux.utils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:08.291 11456 DEBUG neutron.agent.linux.external_process [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:08.292 11456 DEBUG oslo_concurrency.lockutils [req-4671ce34-8d8f-4c13-b7f0-a2cda28e0540 - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:16.351 11456 DEBUG oslo_concurrency.lockutils [req-c42ba5a4-2cd9-4cd4-85da-3904a081b440 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:16.352 11456 DEBUG oslo_concurrency.lockutils [req-c42ba5a4-2cd9-4cd4-85da-3904a081b440 f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:20.207 11456 DEBUG oslo_concurrency.lockutils [req-b644baee-444e-491e-8e76-252e462a2d58 - - - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:20.208 11456 DEBUG oslo_concurrency.lockutils [req-b644baee-444e-491e-8e76-252e462a2d58 - - - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:21.100 11456 DEBUG oslo_concurrency.lockutils [req-b644baee-444e-491e-8e76-252e462a2d58 - - - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:21.102 11456 DEBUG oslo_concurrency.lockutils [req-b644baee-444e-491e-8e76-252e462a2d58 - - - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:40.151 11456 DEBUG oslo_concurrency.lockutils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:40.153 11456 INFO neutron.agent.dhcp.agent [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T20:20:38Z, description=, device_id=fc2bcc3e-8c48-4458-9a15-fc71304b866a, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.4'}], id=2234fc69-f169-4f20-96ae-389f95538c3f, mac_address=fa:16:3e:a8:c0:f1, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=True, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=3, security_groups=[u'4d7ee082-a690-4d8c-9de5-279d04acfd01'], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:39Z
2018-06-27 22:20:40.155 11456 DEBUG neutron.agent.dhcp.agent [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:40.158 11456 DEBUG neutron.agent.linux.dhcp [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:40.160 11456 DEBUG neutron.agent.linux.dhcp [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:40.168 11456 DEBUG neutron.agent.linux.utils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:40.195 11456 DEBUG neutron.agent.linux.dhcp [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:40.198 11456 DEBUG neutron.agent.linux.utils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:40.221 11456 DEBUG neutron.agent.linux.utils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:40.241 11456 DEBUG neutron.agent.linux.utils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:40.243 11456 DEBUG neutron.agent.linux.utils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:40.243 11456 DEBUG neutron.agent.linux.external_process [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:40.244 11456 DEBUG oslo_concurrency.lockutils [req-def4e051-fb67-4a8d-8bab-c2f3ae072b5d f7f7c84e69e14fe4b37ef5eb2ba22e19 cd74b8b050074d2bb18e525e700fe34b - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:43.104 11456 DEBUG oslo_concurrency.lockutils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:43.105 11456 INFO neutron.agent.dhcp.agent [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T20:20:38Z, description=, device_id=fc2bcc3e-8c48-4458-9a15-fc71304b866a, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.4'}], id=2234fc69-f169-4f20-96ae-389f95538c3f, mac_address=fa:16:3e:a8:c0:f1, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=True, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=4, security_groups=[u'4d7ee082-a690-4d8c-9de5-279d04acfd01'], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:41Z
2018-06-27 22:20:43.107 11456 DEBUG neutron.agent.dhcp.agent [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:43.111 11456 DEBUG neutron.agent.linux.dhcp [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:43.116 11456 DEBUG neutron.agent.linux.dhcp [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:43.126 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:43.156 11456 DEBUG neutron.agent.linux.dhcp [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:43.159 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:43.187 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:43.209 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:43.212 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:43.214 11456 DEBUG neutron.agent.linux.external_process [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:43.218 11456 DEBUG oslo_concurrency.lockutils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:44.280 11456 DEBUG oslo_concurrency.lockutils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:20:44.281 11456 INFO neutron.agent.dhcp.agent [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T20:20:38Z, description=, device_id=fc2bcc3e-8c48-4458-9a15-fc71304b866a, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'75db682b-dac0-4e7b-8b92-1cc724ed19bf', u'ip_address': u'11.179.1.4'}], id=2234fc69-f169-4f20-96ae-389f95538c3f, mac_address=fa:16:3e:a8:c0:f1, name=, network_id=83432395-abb8-4ad9-a97c-31272620acc2, port_security_enabled=True, project_id=cd74b8b050074d2bb18e525e700fe34b, revision_number=5, security_groups=[u'4d7ee082-a690-4d8c-9de5-279d04acfd01'], status=DOWN, tags=[], tenant_id=cd74b8b050074d2bb18e525e700fe34b, updated_at=2018-06-27T20:20:43Z
2018-06-27 22:20:44.283 11456 DEBUG neutron.agent.dhcp.agent [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:20:44.285 11456 DEBUG neutron.agent.linux.dhcp [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:20:44.287 11456 DEBUG neutron.agent.linux.dhcp [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:20:44.292 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:44.307 11456 DEBUG neutron.agent.linux.dhcp [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:20:44.308 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:44.323 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:20:44.341 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:44.343 11456 DEBUG neutron.agent.linux.utils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:20:44.344 11456 DEBUG neutron.agent.linux.external_process [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:20:44.345 11456 DEBUG oslo_concurrency.lockutils [req-8c2b5ea6-e2a7-4cd8-b2d2-71d518042116 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:20:48.239 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:20:48.243 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:21:11.627 11456 DEBUG oslo_concurrency.lockutils [req-04242894-e502-45b3-91d9-0e50c00710f4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:21:11.629 11456 DEBUG oslo_concurrency.lockutils [req-04242894-e502-45b3-91d9-0e50c00710f4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:21:13.431 11456 DEBUG oslo_concurrency.lockutils [req-04242894-e502-45b3-91d9-0e50c00710f4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:21:13.432 11456 DEBUG oslo_concurrency.lockutils [req-04242894-e502-45b3-91d9-0e50c00710f4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:21:48.239 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:21:48.243 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:22:48.240 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:22:48.244 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:23:48.240 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:23:48.245 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:24:48.240 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:24:48.245 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:25:18.956 11456 DEBUG oslo_concurrency.lockutils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:25:18.958 11456 DEBUG neutron.agent.dhcp.agent [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:25:18.961 11456 DEBUG neutron.agent.linux.utils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'dhcp_release', 'tap54feb389-26', '11.179.1.4', 'fa:16:3e:a8:c0:f1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:25:18.987 11456 DEBUG neutron.agent.linux.dhcp [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:25:19.000 11456 DEBUG neutron.agent.linux.dhcp [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:25:19.007 11456 DEBUG neutron.agent.linux.utils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:25:19.022 11456 DEBUG neutron.agent.linux.dhcp [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:25:19.024 11456 DEBUG neutron.agent.linux.utils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:25:19.047 11456 DEBUG neutron.agent.linux.utils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:25:19.069 11456 DEBUG neutron.agent.linux.utils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:25:19.071 11456 DEBUG neutron.agent.linux.utils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:25:19.071 11456 DEBUG neutron.agent.linux.external_process [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:25:19.072 11456 DEBUG oslo_concurrency.lockutils [req-18d607b7-2cfd-455f-8b5a-070eec055a51 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:25:48.247 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:25:48.254 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:26:16.386 11456 DEBUG oslo_concurrency.lockutils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:26:16.387 11456 DEBUG neutron.agent.dhcp.agent [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:26:16.390 11456 DEBUG neutron.agent.linux.utils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'dhcp_release', 'tap54feb389-26', '11.179.1.1', 'fa:16:3e:92:3b:35'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:16.416 11456 DEBUG neutron.agent.linux.dhcp [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Building host file: /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 22:26:16.427 11456 DEBUG neutron.agent.linux.dhcp [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Done building host file /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 22:26:16.436 11456 DEBUG neutron.agent.linux.utils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-HUP', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:16.450 11456 DEBUG neutron.agent.linux.dhcp [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Reloading allocations for network: 83432395-abb8-4ad9-a97c-31272620acc2 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 22:26:16.452 11456 DEBUG neutron.agent.linux.utils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-4', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:16.467 11456 DEBUG neutron.agent.linux.utils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2', 'ip', '-6', 'route', 'list', 'dev', 'tap54feb389-26'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:16.487 11456 DEBUG neutron.agent.linux.utils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:26:16.488 11456 DEBUG neutron.agent.linux.utils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:26:16.489 11456 DEBUG neutron.agent.linux.external_process [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:26:16.490 11456 DEBUG oslo_concurrency.lockutils [req-f548b951-eeae-4f4f-9b8e-e80f6073dfd4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:26:47.400 11456 DEBUG oslo_concurrency.lockutils [req-efec7051-0f9d-477e-bd41-95da320cc557 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:26:47.403 11456 DEBUG neutron.agent.dhcp.agent [req-efec7051-0f9d-477e-bd41-95da320cc557 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:26:47.407 11456 DEBUG neutron.agent.linux.utils [req-efec7051-0f9d-477e-bd41-95da320cc557 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-9', '20255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:47.434 11456 DEBUG neutron.agent.linux.utils [req-efec7051-0f9d-477e-bd41-95da320cc557 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 22:26:47.466 11456 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap54feb389-26) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 22:26:47.560 11456 DEBUG neutron.agent.linux.utils [req-03dbff8e-8fea-4532-80ad-6fc943060f81 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:47.574 11456 DEBUG neutron.agent.linux.utils [req-03dbff8e-8fea-4532-80ad-6fc943060f81 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-83432395-abb8-4ad9-a97c-31272620acc2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:47.587 11456 DEBUG oslo_concurrency.lockutils [req-03dbff8e-8fea-4532-80ad-6fc943060f81 - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:26:48.247 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:26:48.249 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:26:57.610 11456 DEBUG oslo_concurrency.lockutils [req-41a33074-2e4e-4752-bac7-9c2bf6491780 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:26:58.017 11456 DEBUG neutron.agent.linux.utils [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:26:58.019 11456 DEBUG neutron.agent.linux.utils [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] Unable to access /var/lib/neutron/external/pids/83432395-abb8-4ad9-a97c-31272620acc2.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:26:58.020 11456 DEBUG neutron.agent.linux.external_process [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:26:58.022 11456 DEBUG neutron.agent.dhcp.agent [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] Calling driver for network: 83432395-abb8-4ad9-a97c-31272620acc2 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 22:26:58.024 11456 DEBUG neutron.agent.linux.utils [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] Unable to access /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:26:58.024 11456 DEBUG neutron.agent.linux.utils [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] Unable to access /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 22:26:58.025 11456 DEBUG neutron.agent.linux.external_process [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] No process started for 83432395-abb8-4ad9-a97c-31272620acc2 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 22:26:58.026 11456 DEBUG neutron.agent.linux.dhcp [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/83432395-abb8-4ad9-a97c-31272620acc2/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-27 22:26:58.027 11456 DEBUG neutron.agent.linux.dhcp [req-63fe8d40-f21f-4bb3-a048-3a5f6d13b6fd - - - - -] No interface exists for network 83432395-abb8-4ad9-a97c-31272620acc2 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-27 22:26:58.070 11456 DEBUG neutron.agent.linux.utils [req-49d0e27e-ab05-41a0-b32c-98b8fa394ca5 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 22:26:58.085 11456 DEBUG neutron.agent.linux.dhcp [req-49d0e27e-ab05-41a0-b32c-98b8fa394ca5 - - - - -] Namespace already deleted: qdhcp-83432395-abb8-4ad9-a97c-31272620acc2 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-27 22:26:58.087 11456 DEBUG oslo_concurrency.lockutils [req-49d0e27e-ab05-41a0-b32c-98b8fa394ca5 - - - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:26:59.517 11456 DEBUG oslo_concurrency.lockutils [req-41a33074-2e4e-4752-bac7-9c2bf6491780 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 22:26:59.519 11456 DEBUG oslo_concurrency.lockutils [req-41a33074-2e4e-4752-bac7-9c2bf6491780 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-83432395-abb8-4ad9-a97c-31272620acc2" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 22:27:10.178 11456 DEBUG neutron.agent.dhcp.agent [req-666dcaa1-51e0-429f-a53b-b2665708435c - - - - -] resync (83432395-abb8-4ad9-a97c-31272620acc2): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-27 22:27:10.180 11456 INFO neutron.agent.dhcp.agent [req-666dcaa1-51e0-429f-a53b-b2665708435c - - - - -] Synchronizing state
2018-06-27 22:27:10.979 11456 INFO neutron.agent.dhcp.agent [req-c44a3436-686d-42c6-9050-41992615660e - - - - -] All active networks have been fetched through RPC.
2018-06-27 22:27:10.981 11456 INFO neutron.agent.dhcp.agent [-] Starting network 6ca5b7c5-23c0-4771-ad3c-ae15aacd634c dhcp configuration
2018-06-27 22:27:10.982 11456 INFO neutron.agent.dhcp.agent [-] Finished network 6ca5b7c5-23c0-4771-ad3c-ae15aacd634c dhcp configuration
2018-06-27 22:27:10.983 11456 INFO neutron.agent.dhcp.agent [req-c44a3436-686d-42c6-9050-41992615660e - - - - -] Synchronizing state complete
2018-06-27 22:27:48.248 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:27:48.252 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:28:48.248 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:28:48.251 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:29:48.248 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:29:48.251 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:30:48.248 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:30:48.251 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 22:31:48.249 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 22:31:48.252 11456 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
